crates/topos-api/src/graphql/checkpoint.rs

```diff
@@ -0,0 +1,44 @@
+use async_graphql::InputObject;
+use serde::{Deserialize, Serialize};
+
+use crate::graphql::errors::GraphQLServerError;
+use topos_uci::{self, SUBNET_ID_LENGTH};
+
+#[derive(Debug, Default, Serialize, Deserialize, InputObject)]
+pub struct CertificateId {
+    pub value: String,
+}
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize, InputObject)]
+pub struct InputSubnetId {
```

Comment:
`InputSubnetId` is the same as `SubnetId` in `certificate` file. We can merge both and have a unique `SubnetId` for input and output

Reply:
We would need to implement `InputObject` and `SimpleObject` by hand then. I thought I leave it like that, since we might have more types in the future like this, so we can have `input` and `output` folders (or sth along these lines)